AzFileDiagnostics/Windows/AzFileDiagnostics.ps1

```diff
@@ -375,6 +375,40 @@ function ValidateLmCompatibilityLevel {
 
 
     
+}
+
+##############################
+#.SYNOPSIS
+# Validate the RestrictSendingNTLMTraffic. 
+#
+#.DESCRIPTION
+# Currently just check the registry key and its value. Check for group policy "Restrict NTLM: Outgoing NTLM traffic to remote servers", if it is not set to 0 or 1, prompt user to change it to expected value. 
+#
+#.EXAMPLE
+#An example
+#
+#.NOTES
+#General notes
+##############################
+function ValidateRestrictSendingNTLMTraffic {
+
+    $RegKeyPath = "HKLM:SYSTEM\CurrentControlSet\Control\Lsa\MSV1_0"
+    $valueName = "RestrictSendingNTLMTraffic"
+
+    $Result = Get-ItemProperty -path $RegKeyPath -Name $valueName -ErrorAction SilentlyContinue
+
+    if ($Result -eq $null) {
+        Write-Log -level success "`n[OK]: HKLM:SYSTEM\CurrentControlSet\Control\Lsa\MSV1_0|RestrictSendingNTLMTraffic IS NOT set, by default it should be 0 (or 1)" 
```

Comment:
In log messages better call out the meaning of 0, 1 and 2 or provide a link where the info can be found. The script will be used by customers so some more context would be helpful for customers to understand and self-mitigate the issue.